Bottom: 1be1919d74a3fbdbee35eceb33e5c829b0840569
Top:    a1c9757586d5ae2af45e801d301014aaead72118
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:23:04 +0000

Refresh of change-found-objects-colour-fr

---

diff --git a/src/action.c b/src/action.c
index 6edfadf..116bb83 100644
--- a/src/action.c
+++ b/src/action.c
@@ -6254,7 +6254,7 @@ ActionUndo (int argc, char **argv, Coord x, Coord y)
 		  ptr2 = (LineType *) ptrtmp;
 	          if (TEST_FLAG (AUTODRCFLAG, PCB))
 		    {
-		      /* undo loses FOUNDFLAG */
+		      /* undo loses CONNECTEDFLAG and FOUNDFLAG */
 		      SET_FLAG(CONNECTEDFLAG, ptr2);
 		      SET_FLAG(FOUNDFLAG, ptr2);
 		      DrawLine (CURRENT, ptr2);
